### DISCUSSION TITLE
[Feature] Feedback while downloading pictures in the app

### DISCUSSION THREAD

**Discussion by fwsmit** [immich-app/immich#8036] [Feature Request]
### The feature

Currently when downloading pictures in the app it will only give confirmation about the download when it's done. When on a slow internet connection this will take a while. It's not clear to the user that the picture is actually downloading.
It would be best if there is a pop up with a progress bar somewhere on screen. You should be able to go to other pictures or anywhere else in the app while downloading 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10100
